basedtyping/runtime_checks.py

```diff
@@ -0,0 +1,20 @@
+from types import UnionType
+from typing import Sequence, cast
+
+
+def issubform(type1: type | UnionType, type2: type | UnionType) -> bool:
+    """``issubclass`` but works with a union as the first arg (like it does for the second arg)
+
+    for example:
+
+    >>> issubclass(int | str, object)
+    TypeError: issubclass() arg 1 must be a class
+    >>> issubtype(int | str, object)
+    True
+    """
+    if isinstance(type1, UnionType):
```

Comment:
What about old union types?

Any any other typeforms maybe `Literal`?

Reply:
it currently only works on types and unions, i think it's out of scope for this change to update it to work on `Literal`s and whatever else as well. i tried that once and fell into a rabbithole. i will add a TODO instead

Reply:
> What about old union types?

i added support for them but they're considered `object` by mypy. i've suppressed the error for now because i don't have a clue how to fix it
![image](https://user-images.githubusercontent.com/57028336/148001358-c513507b-1be7-4966-8be6-6e33c5cf5a4b.png)

Reply:
typeforms are objects